templ_attr.cif

```diff
@@ -677,17 +681,20 @@ save_cromer_mann_coeff
     _type.container              Single
     _type.contents               Real
     _enumeration.def_index_id  '_atom_type.symbol'
-    _units.code                  none
      save_
 
 
 save_hi_ang_fox_coeffs
 
-    _definition.update           2012-11-29
+    _definition.update           2023-06-03
     _description.text
 ;
     The set of data items used to define Fox et al.  coefficients
-     for generation of high angle (s >2.0) X-ray scattering factors.
+     for generation of high angle (2.0 < s < 6.0) X-ray scattering factors.
+
+    f(s) = exp(Sum( c~i~ * s^i^, i=0:3))
+
+    where s = sin(theta)/lambda.
```

Comment:
Similarly, maybe we could also specify here that `s` is measured in reciprocal angstrom (or that lambda is measured in angstrom)?

See, for example, the description of table 6.1.1.5 in ITC G:
```
<...> for the range 2.0 < sin(theta)/lamda < 6.0 A^(-1).
```

Reply:
I'll do lambda is measured in angstrom